src/components/Header.jsx

```diff
@@ -5,14 +5,15 @@ import {
 import PropTypes from 'prop-types';
 import { NextSeo } from 'next-seo';
 import { useSelector } from 'react-redux';
-import UserButton from './UserButton';
-import FeedbackButton from './FeedbackButton';
-import ReferralButton from './ReferralButton';
-import integrationTestConstants from '../utils/integrationTestConstants';
+import UserButton from 'components/UserButton';
+import FeedbackButton from 'components/FeedbackButton';
+import ReferralButton from 'components/ReferralButton';
+import integrationTestConstants from 'utils/integrationTestConstants';
+import { ClipLoader } from 'react-spinners';
 
 const Header = (props) => {
   const {
-    experimentId, experimentData, title, extra,
+    experimentId, experimentData, title, extra, loader,
```

Comment:
Are there any other components apart from `src/pages/data-management/index.jsx` that would benefit from using `loader`?

Reply:
Here are all the usages of this component I could see:
<img width="546" alt="Screenshot 2022-02-04 at 13 37 19" src="https://user-images.githubusercontent.com/65414651/152530293-d7109c6d-8a11-4f8e-b776-419a4a6c6807.png">

Reply:
No, right now I don't think there are other pages that require loader. Usually each component has their own loader.

Reply:
if the loader is not used and won't be used, then why adding it as part for the `Header` component?

Reply:
The consideration to put it here is that it's technically easier to add it to the header and it's clean. We can still keep it on Data Management, but I'd have to add a considerable amount of HTML and CSS to make it look nice and not jerk around. It also opens up an option for the loader to be used by other components in the UI.